tests/layers/test_channelnorm.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from torch import Tensor
+
+from neuralcompression.layers import ChannelNorm2D
+
+
+@pytest.mark.parametrize("affine", [True, False])
+def test_channel_norm_2d(affine, arange_4d_image: Tensor):
```

Comment:
This type of test is not my favorite: the test verifies that the content of the module is the same as the content of the module (since the test replicates the logic of the module).

I would try a more realistic tests: iterate several time on the same input and see that it does normalize ? Or just drop the test.

Reply:
Yeah I didn't like this a ton - but it's more to guard against an implementation change of the module in the future.

Running multiple times won't change anything (unless you want to verify that the mean is still 0 and the variance is still 1). We can think a bit about it.